frontend/src/lib/components/PropertyFilters/PropertyFilter.js

```diff
@@ -110,24 +125,105 @@ export function PropertyFilter({ index, onComplete, logic }) {
                     </Select>
                 </div>
             )}
-            {key && !isOperatorFlag(operator) && (
-                <div className="col-5 pl-0">
+            {displayOperatorAndValue && !isOperatorFlag(operator) && (
+                <div className="col-5 p-0">
                     <PropertyValue
                         type={type}
-                        key={key}
-                        propertyKey={key}
+                        key={propkey}
+                        propertyKey={propkey}
                         operator={operator}
                         value={value}
                         onSet={(value) => {
                             onComplete()
-                            setFilter(index, key, value, operator, type)
+                            setThisFilter(propkey, value, operator, type)
                         }}
                     />
                     {(operator === 'gt' || operator === 'lt') && isNaN(value) && (
                         <p className="text-danger">Value needs to be a number. Try "equals" or "contains" instead.</p>
                     )}
                 </div>
             )}
-        </div>
+        </>
+    )
+}
+
+function CohortPaneContents({ onComplete, setThisFilter, propkey, displayOperatorAndValue }) {
+    const { cohorts } = useValues(cohortsModel)
```

Comment:
`BreakdownFilter.js` also has a Cohort filter, it could be worth extracting that into a component and using that (though in this case without "* All users"). That way you'll also be able to search this Select, which you can't do at the moment

Reply:
It does have a cohort filter but for selecting multiple filters, so a bit of a different use case. Definitely should be searchable though – just made it so.

Reply:
Yep you're absolutely right, good catch